stream/src/main/scala/org/apache/pekko/stream/stage/GraphStage.scala

```diff
@@ -1668,6 +1672,25 @@ trait AsyncCallback[T] {
    * to the invoking logic see [[AsyncCallback#invoke]]
    */
   def invokeWithFeedback(t: T): Future[Done]
+
+  /**
+   * Java API
+   *
+   * Dispatch an asynchronous notification. This method is thread-safe and
+   * may be invoked from external execution contexts.
+   *
+   * The method returns directly and the returned future is then completed once the event
+   * has been handled by the operator, if the event triggers an exception from the handler the future
+   * is failed with that exception and finally if the operator was stopped before the event has been
+   * handled the future is failed with `StreamDetachedException`.
+   *
+   * The handling of the returned future incurs a slight overhead, so for cases where it does not matter
+   * to the invoking logic see [[AsyncCallback#invoke]]
+   */
+  def invokeWithFeedbackCompletionStage(t: T): CompletionStage[Done] = {
+    import pekko.util.FutureConverters._
+    invokeWithFeedback(t).asJava
+  }
```

Comment:
It would be nice if we could split this method into something else, but as the graphdsl is not separated with two `javadsl/scaladsl` , I have to add it here.
cons: 
1. scala developer will see this
2. name is too long.

Reply:
is it possible to add a basic unit test?

Reply:
test added